examples/servicecatalog/principalportfolioassociation.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: servicecatalog.aws.upbound.io/v1beta1
+kind: PrincipalPortfolioAssociation
+metadata:
+  annotations:
+    meta.upbound.io/example-id: servicecatalog/v1beta1/principalportfolioassociation
+    upjet.upbound.io/manual-intervention: "It requires manual intervention user arn"
```

Comment:
Hello @svscheg , have you tried creating a new user managed resource and selecting it?

Reply:
Done, it is work and commited. Uptest passed - https://github.com/upbound/provider-aws/actions/runs/3674681358/jobs/6213195341